src/calibration/table_generator.cpp

```diff
@@ -0,0 +1,61 @@
+#include "table_generator.hpp"
+#include "storage/table.hpp"
+#include "constant_mappings.hpp"
+
+namespace opossum {
+
+  TableGenerator::TableGenerator(std::shared_ptr<TableGeneratorConfig> config) {
```

Comment:
I would rather vote for CalibrationTableGenerator (if that is necessary at all) since we have multiple TableGenerators already. Since this does not generate a table but rather multiple tables, the name isn't a good fit anyways.
Maybe CalibrationSetup or something similar?